guides/csp-configuration.mdx

```diff
@@ -23,15 +23,28 @@ The following CSP directives are used to control which resources can be loaded:
 | Domain | Purpose | CSP directive | Required |
 |:-------|:--------|:--------------|:-------|
 | `d4tuoctqmanu0.cloudfront.net` | KaTeX CSS, fonts | `style-src`, `font-src` | Required |
-| `*.mintlify.dev` | Documentation content | `connect-src` | Required |
+| `*.mintlify.dev` | Documentation content | `connect-src`, `frame-src` | Required |
+| `*.mintlify.com` | Dashboard, API, analytics proxy | `connect-src` | Required |
+| `leaves.mintlify.com` | GitLab webhook integration | `connect-src` | Required |
 | `d3gk2c5xim1je2.cloudfront.net` | Icons, images, logos | `img-src` | Required |
+| `d1ctpt7j8wusba.cloudfront.net` | Mint version and release files | `connect-src` | Required |
+| `mintcdn.com` | Asset tracking, favicons | `img-src`, `connect-src` | Required |
+| `api.mintlifytrieve.com` | Search API | `connect-src` | Required |
+| `mintlify-assets.b-cdn.net` | BunnyCDN assets | `img-src` | Required |
```

Comment:
```suggestion
```

I believe we can remove this one now since we've moved all the assets from here to be served through cloudfront